src/Selenium2Library/keywords/_element.py

```diff
@@ -21,6 +21,13 @@ def __init__(self):
 
     # Public, get element(s)
 
+    def get_webelement(self, locator):
+        """Returns the first WebElement matching the given locator.
+
+        See `introduction` for details about locating elements.
+        """
+        return self.get_webelements(locator)[0]
```

Comment:
Do we want to do this or follow along with what the other find element keywords do? Something like ...

```
        return self._element_find(locator, True, True)
```

where we set the first_only flag to True. I like this a little better if only for the reason that it follows what other find elements are doing and if this changing in the future Get WebElement will do the same thing.